impress-org/givewp
# Feature: add "Make as default" action to the forms list view

Resolves [GIVE-1333]

## Description

This PR adds a "Make as default" action in the forms list table which lives on the forms tab from the campaigns details page.

## Affects



The forms list table.

## Visuals



![image](https://github.com/user-attachments/assets/87e939b6-f7bd-4894-a8bb-8a92701e1989)

## Testing Instructions



1. Go to the "Forms" tab from the Campaigns details page;
2. Make sure to have more than one form;
3. Use the "Make as default" action and make sure everything works as expected.

## Pre-review Checklist






[GIVE-1333]: https://stellarwp.atlassian.net/browse/GIVE-1333?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
